lib/ruby_smb/smb1/packet/trans2/find_first2_request.rb

```diff
@@ -38,7 +42,8 @@ def length
 
           # The Trans2 Data Blcok for this particular Subcommand
           class Trans2Data < BinData::Record
-            smb_gea_list :extended_attribute_list, label: 'Get Extended Attribute List'
+            smb_gea_list :extended_attribute_list, label: 'Get Extended Attribute List',
+              onlyif: -> { parent.trans2_parameters.information_level == FindInformationLevel::SMB_INFO_QUERY_EAS_FROM_LIST}
```

Comment:
Please see the Trans2_Data section of https://msdn.microsoft.com/en-us/library/ee441987.aspx